dlib/dnn/layers_abstract.h

```diff
@@ -2090,6 +2090,53 @@ namespace dlib
     template <typename SUBNET>
     using prelu = add_layer<prelu_, SUBNET>;
 
+// ----------------------------------------------------------------------------------------
+
+    class leaky_relu_
+    {
+        /*!
+            WHAT THIS OBJECT REPRESENTS
+                This is an implementation of the EXAMPLE_COMPUTATIONAL_LAYER_ interface
+                defined above.  In particular, it defines a leaky rectified linear
+                layer.  Therefore, it passes its inputs through the function
+                    f(x) = x>0 ? x : alpha*x
+                where f() is applied pointwise across the input tensor and alpha is a scalar.
```

Comment:
Say `alpha is a non-learned scalar` and then in the prelu docs above change them to say `alpha is a learned scalar`.  Otherwise the docs for these two things are the same.

Reply:
Do you think `alpha` is a good name? I was thinking about something more descriptive, like `slope`?
But again, probably people who are using `leaky_relu` already know what we're referring to.

Reply:
I've modified the `leaky_relu` documentation, but `prelu` already says that `p` is not learned.

Reply:
Oh right, that it does! :)